src/components/collection/Selector/List/CollectionSelectorBody.tsx

```diff
@@ -0,0 +1,141 @@
+                                const row = data[index];
+
+                                return (
+                                    <TableRow
+                                        key={row[COLLECTION_SELECTOR_UUID_COL]}
+                                        component={Box}
+                                        style={style}
+                                        selected={
+                                            row[
+                                                COLLECTION_SELECTOR_UUID_COL
+                                            ] === currentBindingUUID
+                                        }
+                                        hover={selectionEnabled}
+                                    >
+                                        {columns.map((column) => {
+                                            return (
+                                                <TableCell
+                                                    className={`${TABLE_BODY_CELL_CLASS_PREFIX}${column.field}`}
+                                                    align={column.align}
+                                                    key={`${column.field}_${
+                                                        row[
+                                                            COLLECTION_SELECTOR_UUID_COL
+                                                        ]
+                                                    }`}
+                                                    component="div"
+                                                    onClick={
+                                                        Boolean(
+                                                            column.preventSelect
+                                                        )
+                                                            ? undefined
+                                                            : () => {
+                                                                  handleCellClick(
+                                                                      row[
+                                                                          COLLECTION_SELECTOR_UUID_COL
+                                                                      ]
+                                                                  );
+                                                              }
+                                                    }
+                                                >
+                                                    {column.renderCell?.(
+                                                        {
+                                                            row,
+                                                        },
+                                                        filterValue
+                                                    )}
+                                                </TableCell>
```

Comment:
Tried to keep this close to how datagrid works and use the same functions and stuff like that.

Using a `className` here so we don't have to have a lot of repeating styling.